bin/createInhouseSamplesheetFromGS_v2.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```

Reply:
Bovenstaande is zelfde fix voor Python 3 als uit PR van Kim. Uiteraard moet code nog wel getest worden met Python 3....